synapse/storage/stream.py

```diff
@@ -244,6 +244,16 @@ def get_room_events_stream_for_rooms(self, room_ids, from_key, to_key, limit=0,
 
         defer.returnValue(results)
 
+    def get_rooms_that_changed(self, room_ids, from_key):
+        """Given a list of rooms and a token, return rooms where there may have
+        been changes.
+        """
```

Comment:
wouldn't hurt to document the type of the params, especially `from_key`. I always find it impossible to keep track of what type of sync token is what.